source/SimplePicker.js

```diff
@@ -171,6 +171,7 @@
 				this.animateChanged();
```

Comment:
If we ran this function after render, instead of at the beginning of create, it could have the same effect as this fix. What do you think is less costly?